proto/docs/p4runtime.md

```diff
@@ -126,7 +142,22 @@ table_entry {
 
 If `match` for a field is not defined, then the field is treated as don't care
 when constructing the match key for the table entry. Note that leaving match field
-undefined is only allowed for ternary and LPM fields.
+undefined is only allowed for ternary and LPM fields. The following table entry
```

Comment:
The old code allowed for the possibility of don't care range matches - see lines 159-160 of the old code.  Are we back to not allowing don't care range matches?

Reply:
I think that's just an omission